nodewatcher/modules/defaults/network_profile/defaults/wireless.py

```diff
@@ -152,17 +152,33 @@ def get_project_ssid(purpose, attribute='essid'):
 
             # Create mesh interface.
             if 'no-wifi-mesh' not in network_profiles:
-                self.setup_interface(
-                    state,
-                    cgm_models.WifiInterfaceConfig,
-                    parent=wifi_radio,
-                    configuration={
-                        'mode': 'mesh',
-                        'essid': get_project_ssid('mesh'),
-                        'bssid': get_project_ssid('mesh', attribute='bssid'),
-                        'routing_protocols': routing_protocols,
-                    },
-                )
+                if 'wifi-mesh-80211s' in network_profiles:
+                    # Configure 802.11s mesh interface.
+                        self.setup_interface(
```

Comment:
Hm, I did not see that.
The previous indentation that I made was giving me PEP8 errors.
This one does not, but the code does not work.